selfdrive/car/ford/fingerprints.py

```diff
@@ -83,6 +83,7 @@
     ],
     (Ecu.fwdCamera, 0x706, None): [
       b'PJ6T-14H102-ABJ\x00\x00\x00\x00\x00\x00\x00\x00\x00',
+      b'ML3T-14H102-ABR\x00\x00\x00\x00\x00\x00\x00\x00\x00',
```

Comment:
@incognitojam know what the prefix change signifies?

Reply:
ML3T is 2021-22, PJ6T is 2023 and RL3T is 2024. Not sure what the difference between *L3T and *J6T are yet though.

Reply:
I haven't proven it to myself yet, but I believe ML3T-14H102-ABR means <Manufacturer>-<Module>-<Firmware/Revision>.

Reply:
Full table here: https://docs.google.com/spreadsheets/d/1tp6ZYW9fYdDu_xmZWH2Kt39Aw592Ep_7cNJ2kUdPt7M/edit?usp=sharing

`L3T`: F-150 since 2021 and Mach-E. Expedition also uses it in software no (but not hardware part no).
`J6T`: Escape, Expedition, F-150 and Mach-E software and hardware part no

I can't see the pattern